File: 1635-number-of-good-pairs/1635-number-of-good-pairs.cpp
class Solution {
public:
    int numIdenticalPairs(vector<int>& nums) {
        unordered_map<int,int> ump; 
        for(auto it:nums){
            ump[it]++;
        }
        int ans=0;
        for(auto it:ump){
            if(it.second>1){
                ans+=((it.second*(it.second-1))/2);
            }
        }
        return ans;
    }
};

File: 1635-number-of-good-pairs/1635-number-of-good-pairs.java
class Solution {
    public int numIdenticalPairs(int[] nums) {
        int cp = 0 , n;
        n = nums.length;
        for(int i = 0 ; i < n ; i++){
            for (int j = i+1 ; j < n ; j++){
                if(nums[i] == nums[j])
                {
                    cp++;
                }
                
            }
        }
        return cp;
    }
}